src/libraries/System.Text.Json/Directory.Build.props

```diff
@@ -2,8 +2,8 @@
   <Import Project="..\Directory.Build.props" />
   <PropertyGroup>
     <StrongNameKeyId>Open</StrongNameKeyId>
-    <PackageVersion>5.0.1</PackageVersion>
-    <AssemblyVersion Condition="$(TargetFramework.StartsWith('net4'))">5.0.0.1</AssemblyVersion>
+    <PackageVersion>5.0.2</PackageVersion>
+    <AssemblyVersion Condition="$(TargetFramework.StartsWith('net4'))">5.0.0.2</AssemblyVersion>
     <SkipValidatePackage>true</SkipValidatePackage>
```

Comment:
@Anipik is that setting intentional?

Reply:
Found out that this is necessary when incrementing the assembly version for only a subset of supported TargetFrameworks. Unfortunately the servicing docs don't mention that.